frontend/src/scenes/insights/insightDataLogic.ts

```diff
@@ -105,7 +105,10 @@ export const insightDataLogic = kea<insightDataLogicType>([
         propsQuery: [
             () => [(_, props) => props],
             // overwrite query from props for standalone InsightVizNode queries
-            (props: InsightLogicProps) => (props.dashboardItemId?.startsWith('new-AdHoc.') ? props.query : null),
+            (props: InsightLogicProps) =>
+                props.dashboardItemId?.startsWith('new-AdHoc.') || props.dashboardItemId?.startsWith('web-analytics.')
+                    ? props.query
+                    : null,
```

Comment:
Why can't we use `new-AdHoc.` for web analytics as well? Don't think it would be great to add more special handling like this and we'd probably need to adapt more places.

Reply:
Hmm - it just seemed worse overall to lie about what kind of insight that it is, as in `new-AdHoc` doesn't accurately describe an insight that's hosted on the web analytics dashboard.

Reply:
Also not a big fan of adding more special cases. Perhaps sticking to a prefix like `new-Web.foo` will require the least special handling?

Reply:
I just reverted that change